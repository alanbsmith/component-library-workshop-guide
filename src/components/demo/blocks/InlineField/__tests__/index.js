import { generateSnapshots } from "__tests__/helpers";

import InlineField from '../index';

generateSnapshots(InlineField);
