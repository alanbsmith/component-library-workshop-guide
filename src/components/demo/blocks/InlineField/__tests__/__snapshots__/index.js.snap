// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 0.25rem;
  border: solid 1px #ecedf0;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  padding: 0.75rem;
  -webkit-transition: all 200ms ease;
  transition: all 200ms ease;
}

.c0:hover {
  border-color: #c0c4cb;
}

<div
  className="c0"
/>
`;

exports[`Container should apply warning modifier 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 0.25rem;
  border: solid 1px #ecedf0;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  padding: 0.75rem;
  -webkit-transition: all 200ms ease;
  transition: all 200ms ease;
  border-color: #967100;
}

.c0:hover {
  border-color: #c0c4cb;
}

.c0:hover {
  border-color: #967100;
}

<div
  className="c0"
/>
`;
